tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/errors/TableNotFoundError.java

```diff
@@ -0,0 +1,20 @@
+package com.google.cloud.bigquery.utils.queryfixer.errors;
+
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.utils.queryfixer.entity.Position;
+
+import lombok.Getter;
+
+/** A class to represent the "Table Not Found" errors from BigQuery. The errors are presented in this form:
+ * "Not found: Table [TableName] was not found", where TableName is the incorrect table name. */
+@Getter
+public class TableNotFoundError extends BigQuerySemanticError {
+
+  String incorrectTableName;
```

Comment:
nit: semantically it is clear the class is used to represent the errors, so you can just use "String tableName", "incorrectTableName" looks a bit redundant.